papas/graphtools/DirectedAcyclicGraph.h

```diff
@@ -222,7 +222,7 @@ void BFSVisitor<N>::traverse(const Nodeset<N>& nodes, typename DAG::enumVisitTyp
   std::queue<int> nodeDepth;  // keeps track of the node depths so we can limit how deep we go if we wish
 
   // Mark the current node as visited and enqueue it
-  for (auto const& node : nodes) {
+  for (auto& node : nodes) {
```

Comment:
Yes, just got myself confused for a moment. Sorry. Have fixed so will arrive in a later pull request.